src/generate.ts

```diff
@@ -32,7 +33,11 @@ export function generate(src: string = conf.apiFile, dest: string = conf.outDir)
   }
 
   const header = processHeader(schema);
-  const config: Config = {header, dest};
+  const config: Config = {
+    header,
+    dest,
+    baseUrl: `${(schema.schemes || [])[0] || 'http'}://${schema.host}${schema.basePath}`,
```

Comment:
I would prefer simple `if` construction (`if (schema.schemes && schema.schemes.length) ... else ...`), these `||` expressions are a bit hard to read.